sys-apps/less/less-650.recipe

```diff
@@ -48,7 +49,7 @@ BUILD_PREREQUIRES="
 BUILD()
 {
 	make -f Makefile.aut distfiles
-	runConfigure ./configure --with-editor=nano \
+	runConfigure ./configure --prefix=$prefix --with-editor=nano \
```

Comment:
prefix not needed here, that's done by runConfigure, please use commandBinDir here.